src/inc/DefaultSettings.h

```diff
@@ -24,7 +24,7 @@ constexpr COLORREF DEFAULT_BACKGROUND = COLOR_BLACK;
 constexpr COLORREF DEFAULT_BACKGROUND_WITH_ALPHA = OPACITY_OPAQUE | DEFAULT_BACKGROUND;
 constexpr short DEFAULT_HISTORY_SIZE = 9001;
 const std::wstring DEFAULT_FONT_FACE { L"Consolas" };
-constexpr int DEFAULT_FONT_SIZE = 12;
+constexpr int DEFAULT_FONT_SIZE = 10;

```

Comment:
?

Reply:
Oh, that's a leftover. Probably something that should be added to #869. 12 **point** font is kinda huge, while 12px is fine.